nixos/modules/services/backup/burpClient.nix

```diff
@@ -0,0 +1,65 @@
+{ config, lib, networking, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.burpClient;
+
+in
+
+{
+
+  ###### interface
+
+  options = {
+
+    services.burpClient = {
+
+      enable = mkOption {
+        default = false;
+        description = "Whether to enable burp.";
+      };
+
+      configFile = mkOption {
+        default = "/etc/burp/burp.conf";
```

Comment:
@jagajaga, I understand the reason for that, but burp has some files referred to in the configuration file that are typically in the same directory /etc/burp like a certificate authority, some scripts and various client configuration files.
While I intend to move them I simply do not have to time and knowledge with nix to do it now.